addons/token-exchange/agent_mirrorpeer_controller_test.go

```diff
@@ -37,9 +38,7 @@ var (
 					},
 				},
 			},
-			SchedulingInterval:    "10m",
-			ReplicationSecretName: "my-super-secret-credentials",
-			Mode:                  "journal",
+			SchedulingInterval: []string{"10m", "5m", "30m", "1h"},
```

Comment:
add tests when list contains same values, or no values.

Reply:
I have added duplicate values in this but i don't see keeping an empty list as it won't enter the loop in anycase.